xmake/modules/package/manager/vcpkg/install_package.lua

```diff
@@ -80,12 +78,8 @@ function _install_for_manifest(vcpkg, name, opt)
     -- init triplet
     local arch = opt.arch
     local plat = opt.plat
-    if plat == "macosx" then
-        plat = "osx"
-    elseif plat == "bsd" then
-       plat = "freebsd"
-    end
-    arch = configurations.arch(arch)
+    plat = configurations.plat(plat)
+	arch = configurations.arch(arch)
```

Comment:
here

Reply:
already fixed in https://github.com/xmake-io/xmake/pull/2773/commits/b65703e6b570c00629f8f2431ca329c286290c9c